src/components/orders/OrdersUserDetailsTable/index.tsx

```diff
@@ -22,6 +23,45 @@ const Wrapper = styled(StyledUserDetailsTable)`
   > tbody > tr {
     grid-template-columns: 12rem 7rem repeat(2, minmax(16rem, 1.5fr)) repeat(2, minmax(18rem, 2fr)) 1fr;
   }
+
+  .header-title {
+    display: none;
+  }
+
+  @media ${MEDIA.mobile} {
+    > thead > tr {
+      display: none;
+    }
+    > tbody > tr {
+      border: 0.1rem solid rgb(151 151 184 / 10%);
```

Comment:
Is there any color in the theme you could use for that?
In case not, could you add it there?

Reply:
Yes, please use existing colors:
- Check if the color exists in the theme/variables
- IF not > check if this transparent color is a derrivative of an existing color but just with an opacity (10%) > use the `polished` library to use the `transparentize` method on the **existing** color.
- New/unique color? --> Add to theme/variables.